python/postprocessing/modules/btv/btagSFProducer.py

```diff
@@ -164,8 +201,12 @@ def __init__(self, era, algo = 'csvv2', sfFileName = None, verbose = 0):
         self.branchNames_central_and_systs = {}
         for central_or_syst in self.central_and_systs:
             if central_or_syst == "central":
+                if era.find("FastSim"):
```

Comment:
This looks problematic usage of find function. find() will return -1 if not find, which will still pass the if statement (using 0 and non0 I believe in python). A simple "FastSim" in era statement as above is more correct.